src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -69,14 +69,20 @@ public class NodeController : Controller
         /// <summary>An interface implementation for the blockstore.</summary>
         private readonly IBlockStore blockStore;
 
-        public NodeController(IFullNode fullNode, ILoggerFactory loggerFactory,
-            IDateTimeProvider dateTimeProvider, IChainState chainState,
-            NodeSettings nodeSettings, IConnectionManager connectionManager,
-            ConcurrentChain chain, Network network, IPooledTransaction pooledTransaction = null,
-            IPooledGetUnspentTransaction pooledGetUnspentTransaction = null,
+        public NodeController(
```

Comment:
Whats all the changes in this class? just a reorder?

Reply:
Yeah just to reorder 👍